loops-and-arrays.js

```diff
@@ -13,7 +13,11 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
 
-
+console.log(names)
+for (let i = 0; i < names.length; i++) {
+    names[i] += "je";       // names[i] = names + "je"; (werkt niet)
```

Comment:
Betreft je comment: rechts van de '=' verwijs je naar de hele array en niet naar de index (enkele naam). Wat wel werkt is: 'names[i] = names[i] + "je";